README.md

```diff
@@ -274,101 +271,27 @@ To run Darknet on Linux use examples from this article, just use `./darknet` ins
 
 ### How to compile on Windows (using `CMake`)
 
-This is the recommended approach to build Darknet on Windows if you have already
-installed Visual Studio 2015/2017/2019, CUDA >= 10.0, cuDNN >= 7.0, and
-OpenCV >= 2.4.
-
-Open a Powershell terminal inside the cloned repository and launch:
-
-```PowerShell
-.\build.ps1
-```
-
-### How to compile on Windows (using `vcpkg`)
+This is the recommended approach to build Darknet on Windows.
 
-1. Install or update Visual Studio to at least version 2017, making sure to have it fully patched (run again the installer if not sure to automatically update to latest version). If you need to install from scratch, download VS from here: [Visual Studio Community](http://visualstudio.com)
+1. Install Visual Studio 2017 or 2019. In case you need to download it, please go here: [Visual Studio Community](http://visualstudio.com)
 
-2. Install CUDA
+2. Install CUDA (at least v10.0) enabling VS Integration during installation.
 
-3. Install [vcpkg](https://github.com/Microsoft/vcpkg) and try to install a test library to make sure everything is working, for example `vcpkg install opengl`
-
-4. Open Powershell and type these commands:
+3. Open Powershell and type these commands:
 
 ```PowerShell
-PS \>                  cd vcpkg
-PS Code\vcpkg>         .\vcpkg install darknet[full]:x64-windows #replace with darknet[opencv-base,weights]:x64-windows for a quicker install; use --head if you want to build latest commit on master branch and not latest release
+PS Code\>              git clone https://github.com/microsoft/vcpkg
+PS Code\>              cd vcpkg
+PS Code\vcpkg>         $env:VCPKG_ROOT=$PWD
+PS Code\vcpkg>         .\bootstrap-vcpkg.bat
+PS Code\vcpkg>         .\vcpkg install darknet[full]:x64-windows #replace with darknet[opencv-base,cuda,cudnn]:x64-windows for a quicker install of dependencies
+PS Code\vcpkg>         cd ..
+PS Code\>              git clone https://github.com/AlexeyAB/darknet
+PS Code\>              cd darknet
+PS Code\darknet>       .\build.ps1
 ```
```

Comment:
`scripts/setup.ps1` might be what you are requesting?

Reply:
@cenit Yes, sure, thanks! May be better to place `scripts/setup.ps1` and `scripts/setup.sh` to the root directory?
If there are too many files in the root directory, it may be better to delete the files: `image_yolov3.sh, net_cam_v3.sh, video_yolov3.sh`

Reply:
it's just that those scripts are not well tested, so I am a little bit afraid of exposing them in the root folder...

Reply:
Ok, lets keep it in the scripts folder for a while.